data/115/885/753/1/1158857531.geojson

```diff
@@ -10,6 +10,8 @@
     "geom:latitude":59.923461,
     "geom:longitude":30.213247,
     "iso:country":"RU",
+    "label:eng_x_preferred_disambiguation":"Saint Petersburg City",
```

Comment:
Expecting these to be lists https://github.com/whosonfirst/whosonfirst-properties/tree/main/properties/label